pkg/config/config_test.go

```diff
@@ -30,7 +30,7 @@ attribution:
 
 		require.NoError(t, os.WriteFile(configFilePath, []byte(fileContents), 0600))
```

Comment:
We should add a test when `./sauced.yaml` doesn't exist, but `~/.sauced.yaml` does.

Reply:
> We should add a test when ./sauced.yaml doesn't exist, but ~/.sauced.yaml does.

This is essentially the test I removed in: https://github.com/open-sauced/pizza-cli/pull/146 - in the case that this runs and the developer doesn't have a `~/.sauced.yaml`, a simple test would fail. If we attempt to stub one for the user, this could conflict with people who _do_ have a `~/.sauced.yaml` - there's alot of tricky scaffolding and edge cases for people running these tests with or without a home dir `~/.sauced.yaml`. Truly isolating this test probably needs to run in a container as e2e suites.

In the future, might be worth looking at Dagger since their CI/CD approach would solve this use case (running tests / builds / etc. in isolated environments within containers that can pass artifacts to eachother)

TLDR: we should punt on more complex testing that requires abunch of scaffolding

Reply:
Sounds good. Forgot about that. Thanks for the reminder.